patch/node/node_wrapper.h

```diff
@@ -51,6 +50,7 @@ struct _JSValue {
 };
 
 typedef struct _JSValue JS_Value;
+typedef struct _JSValue JS_Result;
```

Comment:
Agreed, this shouldn't be necessary. I never understood the reason for the JXResult typedef in JXCore. It only causes confusion because it's just another name for JXValue.

I see the bridge currently uses JXResult in a couple places, but that should just be updated to JXValue/JS_Value.

---
In reply to: [73125188](https://github.com/openT2T/node-apphost/pull/7#discussion_r73125188) [](ancestors = 73125188)